mos_tests/functions/os_cli.py

```diff
@@ -119,3 +149,55 @@ class Ironic(CLICLient):
 
 class Murano(CLICLient):
     command = 'murano'
+
+
+class Swift(CLICLient):
+    command = 'swift'
+
+    def post(self, name):
+        return self('post {0}'.format(name))
+
+    def list(self, name=''):
+        output = self('list -l', params=name)
+        return output.split('\n')
+
+    def delete(self, name, filename=None):
+        params = name
+        if filename:
+            params += ' {0}'.format(filename)
+        return self('delete', params=params)
+
+    def upload(self, name, filename, option=None):
+        if option:
+            return self('upload {0} {1} {2}'.format(name, option, filename))
+        else:
+            return self('upload {0} {1}'.format(name, filename))
+
+
+class S3CMD(CLICLient):
+    command = 's3cmd'
+
+    def bucket_make(self, name):
+        return self('mb s3://{0}'.format(name))
+
+    def bucket_ls(self, name=None):
+        params = ''
+        if name:
+            params += 's3://{0}'.format(name)
+        output = self('ls', params=params)
+        return output.split('\n')
```

Comment:
Each time after using this method, you join result list back to string. Can't you provide an example of output?